sbol_utilities/component.py

```diff
@@ -641,7 +692,7 @@ def digestion(reactant:sbol3.Component, restriction_enzymes:List[sbol3.Externall
     for re in restriction_enzymes:
         enzyme = Restriction.__dict__[re.name]
         restriction_enzymes_pydna.append(enzyme)
-        assembly_plan.features.append(re)
+        #assembly_plan.features.append(re)
```

Comment:
Now its added by the assembly plan. When I run the assembly plan the first time and because digeastion is very likely to be in a for loop it tries to add the same restriction enzyme to the doc causing errors.

Reply:
The you should remove the code, rather than commenting it out. The code will still be there in the history if you ever need it back.

Reply:
True, sometimes I test things commenting instead of deleting and forget about them. I deleted all the commented code that I found.